src/content/developers/tutorials/monitoring-geth-with-influxdb-and-grafana/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Monitoring Geth with InfluxDB and Grafana
+description: 
+author: "Mario Havel"
+tags: ["clients", "geth", "nodes"]
+skill: intermediate
+lang: en
+sidebar: true
+---
+
+Monitoring your node is a best practice which can help you to be informed about its performance, status, identify potential issues and even alert you if some problem occurs. Client implementations including [Geth](https://geth.ethereum.org/) offer thorough metrics collection and reporting. 
```

Comment:
This tutorial will help you set up monitoring for your Geth node so you can better understand its performance and identify potential problems.